### PR TITLE
Custom function support

### PR DESCRIPTION
**Pull request by jkelvie**
Ability to pass a function to VirtualAlexa, instead of just a file/handler name

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,4739 @@
...
+      "integrity": "sha1-DM8tiRZq8Ds2Y8eWU4t1rG4RTZ0=",
+      "dev": true
+    },
+    "lodash._root": {
+      "version": "https://registry.npmjs.org/lodash._root/-/lodash._root-3.0.1.tgz",
+      "integrity": "sha1-+6HEUkwZ7ppfgTa0YJ8BfPTe1pI=",
+      "dev": true
+    },
+    "lodash.escape": {
+      "version": "https://registry.npmjs.org/lodash.escape/-/l
```
Comment (jperata) [inline package-lock.json]: this is the version that had the vulnerability reported on BST, removing node_modules, package-json.lock and doing npm install again should bring the version 0.3.9

Comment (jkelvie) [reply package-lock.json]: I have not checked in package-json.lock though - should I?

Comment (jkelvie) [reply package-lock.json]: Okay, I take that back - I guess I added it as part of this PR - do we want this in the repo? I believe the answer is yes - but let me know your thoughts. And if so, I will fix it as you recommend.

Comment (jperata) [reply package-lock.json]: Yes, i agree that is good that the lock stays.

**Pull request merged**